helm/chart-operator-resource-chart/Chart.yaml

```diff
@@ -1,3 +1,3 @@
 name: chart-operator-resource-chart
-version: 0.1.0
+version: 0.3.0
```

Comment:
Just for my curiosity. Why is it 3 now? What does this mean? Do we have to bother?

Reply:
Yes, now that we are not adding `[[ .SHA ]]` to the chart versions we need to bump those versions when there are changes in the charts, so that the instances available on the different channels don't get overridden. In this case, I needed to bump twice because i manually pushed `0.2.0` during testing, this won't be the usual case.

Reply:
I looked into plugins for this some time ago, looks like as of today this is the better alternative https://github.com/mbenabda/helm-local-chart-version. I'll give it a try, maybe we can do an automated version bump on CI if some of the chart files have changed, but this would require an additional commit. Need to check.